Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5132.cs

```diff
@@ -44,7 +44,7 @@ protected override void Init()
 			});
 		}
 
-		static string DefaultFontFamily()
+		public static string DefaultFontFamily()
```

Comment:
is making this public and reusing with 9440 necessary?

Reply:
I'm using this code into the Issue9440 UItest